work_lights.py

```diff
@@ -35,35 +36,31 @@
 
 def TurnOnLights(lights):
   try:
-    lights(1, 'state')
-    lights(2, 'state')
+    [lights(light, 'state') for light in LIGHTS]
   except QhueException as err:
     if LOGGING:
       print('Turning lights on')
-    lights(1, 'state', on=True)
-    lights(2, 'state', on=True)
+    [lights(light, 'state', on=True) for light in LIGHTS]
 
 
 def TurnOffLights(lights):
-  lights(1, 'state', on=False)
-  lights(2, 'state', on=False)
+  [lights(light, 'state', on=False) for light in LIGHTS]
 
 
 def SetAmbientColor(lights):
   x = round(random.random(), 3)
   y = round(random.random(), 3)
 
-  lights(1, 'state', xy=[x, y], bri=254, transitiontime=100)
-  lights(2, 'state', xy=[x, y], bri=254, transitiontime=100)
-
+  [lights(light, 'state', xy=[x, y], bri=254, transitiontime=100)
+          for light in LIGHTS]
   if LOGGING:
     print('x={}, y={} @ {}'.format(x, y, datetime.datetime.now()))
   return
 
 
 def SetGVCColor(lights):
-  lights(1, 'state', xy=[0.300, 0.300], bri=254)
-  lights(2, 'state', xy=[0.300, 0.300], bri=254)
+  [lights(light, 'state', xy=[0.300, 0.300], bri=254)
```

Comment:
I'm for the change. The only observation is that if you wanted the lights to be different colors, you'd have to individually set their state.